### DISCUSSION TITLE
Change "Entitäten verfügbar gemacht" to "freigegebene Entitäten" for German i18n

### DISCUSSION THREAD

**Discussion by bannert1337** [home-assistant/frontend#21166] [Localization] [answered]
![image](https://github.com/home-assistant/frontend/assets/58707896/6f2b230f-82ab-4ef7-971a-60a5af4c274b)
Right now, this button looks really big and offputting. It would reduce complexity and increase visibility and comprehensibility to change it to "freigegebene Entitäten" instead of "Entitäten verfügbar gemacht".
It could look like this:
![image](https://github.com/home-assistant/frontend/assets/58707896/f543dd94-c8b4-4ce5-be1e-7caa7ff0752a)

**Comment by karwosts** [comment]
There are directions here for how to change translations:

https://developers.home-assistant.io/docs/translations/

**Comment by NoRi2909** [comment]
@bannert1337  Sorry for the late comment on this issue here. The problem here is that nobody has come up with a good _and_ short German translation for "expose" so far that does not interfere with other translations.

Currently it's "verfügbar machen" which is a pretty good match for "to expose" but unfortunately bit on the longer side …

We cannot use "freigeben" here as this is already the established German translation of "to share" in many places and across many computer and mobile OSes, too.

I'm open to suggestions, but note that there are about 30 strings containing "expose" in one or the other form that all need to match here!

Simple problem, complex solution …

**Comment by NoRi2909** [comment, marked as answer]
With the January release the button is now just `xx verfügbare Entitäten`, That's not 100% accurate, unless you see it from the voice assistant's side, but as the word is consistent this should not cause any confusion here:

![image](https://github.com/user-attachments/assets/65350665-5cf5-4633-bfd7-816a66f4ecad)

This solves the issue of the former button label being overly long so it did not fit a single line (on a phone e.g.) along with the `xx Assist-Gerät(e)` that shows up when you have one of those (see above).
